config/helpers.js

```diff
@@ -0,0 +1,25 @@
+const path = require('path');
```

Comment:
You can use something like:

```
new HtmlWebpackPlugin({
...
   chunks: [
      'commons',
      'vendors',
      'app',
    ],
```

[See](https://github.com/jantimon/html-webpack-plugin/issues/140)

So, I think this file it's not needed